### DISCUSSION TITLE
自定义顶栏无法自动刷新历史记录，这是bug还是特性？

### DISCUSSION THREAD

**Discussion by silv3rarr0w** [the1812/Bilibili-Evolved#4948] [提问 / Q&A]

**Comment by Geeyun-JY3** [comment]
历史应该是会自动刷新的：<https://github.com/the1812/Bilibili-Evolved/issues/1311#issuecomment-743978107>，不过最近实测的确无法自动刷新，可能是 bug？

**Reply by the1812** [reply]
应该是 v2 的 bug, 没给历史加刷新

**Reply by the1812** [reply]
开始写的时候发现是因为 v2 的历史支持翻页所以不能自动刷新了

**Reply by Geeyun-JY3** [reply]
目前实测切换 tab 可以手动刷新，不过这样切换到别的 tab 再切换回来的手动刷新方式感觉略显麻烦。是否考虑增加其他手动刷新方式？

**Reply by the1812** [reply]
加个手动刷新的按钮比较合适
